crm_lead_stage_monitoring/crm.py

```diff
@@ -35,9 +35,10 @@ def write(self, cr, uid, ids, vals, context=None):
             vals['last_activity_stage'] = fields.datetime.now()
         return super(CrmLead, self).write(cr, uid, ids, vals, context=context)
 
-    def copy(self, cr, uid, id, default=None, context=None):
+    def copy(self, cr, uid, rec_id, default=None, context=None):
```

Comment:
I don't remember the thread exactly but I think there has been some questioning if we could change the name of id in the signature of default ORM methods because of some introspection happening in the core expecting just "id". I think this was discussed when somebody wanted to test for variable named "id" in the automatic tests in the OCA. Am I wrong with this or has the debate been closed since? Anybody remember?  @pedrobaeza @StefanRijnhart @bwrsandman any idea?

PR looks LGTM other than this.